app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -280,6 +280,7 @@ class NavigationOverlayFragment : Fragment() {
 
                         val settings = Settings.getInstance(context!!)
                         if (settings.shouldShowFxaOnboarding()) {
+                            serviceLocator.screenController.showNavigationOverlay(fragmentManager, true)
```

Comment:
With the current implementation, I think this is the right way to do this. However, I just want to mention that ideally this and `showFxaOnboardingScreen` would actually be called from the callback we get for login success – namely, the function that calls `finishAuthentication...` in `FxaRepo` – so we don't have to maintain "did we see onboarding?" state. Perhaps in a future refactor. :)